frontend/javascripts/oxalis/api/api_loader.ts

```diff
@@ -62,4 +62,6 @@ class Api {
   }
 }
 
+export type ApiType = Api;
```

Comment:
in the request and model module, I used `typeof $instance` and here I'm using the class directly. Should both be fine :) It was important to me that the tests do an `import type` so that mocking is not disturbed.